src/style.css

```diff
@@ -0,0 +1,69 @@
+@font-face {
+  font-family: Avenir Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+@font-face {
+  font-family: Avenir Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+.bar--1 {
+  height: 70px;
+  width: 900px;
```

Comment:
height is okay but do not hard code width, to make it responsive you can 
`width: 100%;`